### DISCUSSION TITLE
检测模型传图的宽高比例大概是多少比较合适？

### DISCUSSION THREAD

**Discussion by Alanhzl** [PaddlePaddle/PaddleOCR#13634] [Q&A] [answered]
![origin](https://github.com/user-attachments/assets/b4c5efc1-3f93-4526-81dc-852a50999db1)
如图，原始图的宽高为1707*151，此时检测出多个检测框，大致上以标点划分，但是这样会导致漏个别字或标点符号。
![result](https://github.com/user-attachments/assets/bfb0437c-1fb4-41f3-92ff-144312e225a7)
后面我把高度翻倍，填充白色，就能检测出完整的一整行。

问题：v3检测模型的宽高比例大概多少是合适的？上面的只是简单粗暴翻倍高度，想问问有没有合适的比例

**Comment by UserWangZz** [comment, marked as answer]
没有具体的比例，按经验调整即可
